lib/std/c/freebsd.zig

```diff
@@ -5,12 +5,21 @@ const maxInt = std.math.maxInt;
 const iovec = std.os.iovec;
 const iovec_const = std.os.iovec_const;
 
+pub const CPU_SETSIZE = 256;
+pub const cpuset_t = extern struct {
+    __bits: [CPU_SETSIZE / (@sizeOf(c_long) * 8)]c_long,
+};
```

Comment:
copying the os macro calc lets `CPU_SETSIZE` work with non-multiples of `64` (bits per long):
```zig
__bits: [(CPU_SETSIZE + (@bitSizeOf(c_long) - 1)) / @bitSizeOf(c_long)]c_long
```